dynatrace-aws-logs.sh

```diff
@@ -53,14 +53,18 @@ case $MAIN_OPTION in
 
   function print_help_deploy {
     printf "
-usage: dynatrace-aws-logs.sh deploy --target-url TARGET_URL --target-api-token TARGET_API_TOKEN [--require-valid-certificate {true|false}] [--stack-name STACK_NAME]
+usage: dynatrace-aws-logs.sh deploy --target-url TARGET_URL --target-api-token TARGET_API_TOKEN --use-existing-active-gate {true|false} [--target-paas-token TARGET_PAAS_TOKEN] [--require-valid-certificate {true|false}] [--stack-name STACK_NAME]
 
 arguments:
   -h, --help            show this help message and exit
   --target-url TARGET_URL
-                        The URL to Your Dynatrace SaaS environment logs ingest target. Currently only ActiveGate endpoint is supported: https://<active_gate_address>:9999/e/<environment_id> (e.g. https://22.111.98.222:9999/e/abc12345)
+                        The URL to Your Dynatrace SaaS environment logs ingest target. If you choose new ActiveGate deployment (--use-existing-active-gate=false), provide Tenant URL (https://<your_environment_ID>.live.dynatrace.com). If you choose to use existing ActiveGate (--use-existing-active-gate=true), provide ActiveGate endpoint: https://<active_gate_address>:9999/e/<environment_id> (e.g. https://22.111.98.222:9999/e/abc12345).
```

Comment:
I like Azure docs wording (https://www.dynatrace.com/support/help/shortlink/azure-log-fwd#dynatrace-) + I'd go with true first, rather then false, so maybe something in this direction:
```
If you want to use an existing ActiveGate (--use-existing-active-gate=true), set it to your ActiveGate endpoint: https://<your_activegate_IP_or_hostname>:9999/e/<your_environment_ID> If you want to deploy new ActiveGate in AWS EC2 (--use-existing-active-gate=true), set it to your Dynatrace SaaS environment: https://<your_environment_ID>.live.dynatrace.com
```

Reply:
Check out the docs
https://bitbucket.lab.dynatrace.org/projects/WX/repos/doc/pull-requests/6090/overview
We are going with something like this 
![image](https://user-images.githubusercontent.com/16317514/122564427-9ec58e00-d045-11eb-8f77-40f6c28601a9.png)
So we kinda push the option with new AG to the top, as it's the easy way to do it (targeting non advanced user that might not have private AG). 

So we can either
1) Discuss ordering in docs and possibly change if you see it other way
2) Change this wording here and have this little order mixing
How do you see it?

Reply:
I try to be consistent with Azure; personally, I would prefer different params altogether, like 
'--tenant-url' if you want AG or '--active-gate-url' if you want to use existing 
and get rid of this --use-existing-active-gate flag which is like "main controller" for params right now

Reply:
True, I also just noticed in the docs PR that new AG goes first. Personally, different order in different places makes sense to me, as I don't expect somebody going from both doc and usage at the same time. But we can go with doc order.

I won't say consistency doesn't hurt me here a bit, too, but let's have it same as Azure;)

But I'm thinking, since this is formatted as is, can't we make `If true` `If false` for both `--target-url` and `--use-existing-active-gate` and go begin each from a new line, rather then in a single line? I don't mind the ordering at all then.

Reply:
I just updated this, please check. I think it is more readable now with clean separated cases - each in new line